apollo-router/tests/type_conditions.rs

```diff
@@ -431,3 +486,43 @@ query Search($movieResultParam: String, $articleResultParam: String) {
       }
     }
 }"#;
+
+fn normalize_response_extensions(mut response: Response) -> Response {
+    let extensions = &mut response.extensions;
+
+    for (key, value) in extensions.iter_mut() {
+        visit_object(key, value, &mut |key, value| {
+            if key.as_str() == "operation" {
+                if let Value::String(s) = value {
+                    let new_value = Document::parse(s.as_str(), key.as_str())
+                        .unwrap()
+                        .to_string();
```

Comment:
Maybe `.serialize().no_indent().to_string()` here?

Reply:
Great call! fixed in [8fb3650](https://github.com/apollographql/router/pull/5808/commits/8fb3650c834419cf424591591cd78dadb4f9763e)